src/test/resources/catalog_2.ttl

```diff
@@ -150,23 +152,38 @@
         dcat:theme                      <http://publications.europa.eu/resource/authority/data-theme/ENVI> , <http://publications.europa.eu/resource/authority/data-theme/GOVE> ;
         dqv:hasQualityAnnotation        [ a                dqv:QualityAnnotation ;
                                           dqv:inDimension  <http://iso.org/25012/2008/dataquality/Currentness> ;
-                                          prov:hasBody     [ rdf:value  "Denne teksten sier noe om aktualiteten. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras mattis consectetur purus sit amet fermentum. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus."@no ]
+                                          oa:hasBody     [ a    oa:TextualBody ;
+                                            rdf:value       "Denne teksten sier noe om aktualiteten. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Cras mattis consectetur purus sit amet fermentum. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus." ;
+                                            dc:language     "no" ;
```

Comment:
Eksempelet i spekken som viser bruk av oa:TextualBody er oppdatert med korrekt range for language og format:
```
:aTextBody
   a oa:TextualBody ;
      rdf:value “Statistisk sett mangler det 1,23% av objektene i datasettet” ;
      dct:language <http://publications.europa.eu/resource/authority/language/NOB> ;
      dct:format <http://publications.europa.eu/resource/authority/file-type/TXT> .
```
https://informasjonsforvaltning.github.io/dqv-ap-no/#Kvalitetsnote_harTekstdel

Oppsummert: du må
- endre til dct:language og bruke lenke til eu-vokabularet, og
- endre verdien til format til eu-vokabularet

som vist i eksempelet.

Reply:
605e993